script.js

```diff
@@ -1,12 +1,44 @@
-let colorlist = ['gold', 'yellow', 'turquoise', 'red']
+var imgs = [];
+var d = 0;
+let img;
```

Comment:
I don't think this variable is being used anywhere any more, so you can remove it